### PR TITLE
ci: run builds on Go 1.9

### PR DESCRIPTION
**Pull request by ttaylorr**
This pull request upgrades our CI runners to build and test LFS on Go 1.9. 

I want to wait on merging this for a few days to either:

1. Wait for any lingering bugs from the latest release to be resolved, and
2. Wait for Chocolatey to upgrade to Go 1.9.

---

/cc @git-lfs/core

### CODE REVIEW

**Comment by sschuberth** [comment]
FYI, I've asked for an upgrade of the Chocolately package at https://github.com/carstn/chocolatey-packages/issues/19.

**Comment by suthineek** [comment]
V2.4.0

**Comment by ttaylorr** [comment]
I'm resurrecting this. The last that I recall, this had an issue building on Windows under Go 1.9, but I'm hoping that bumping to 'master' and Go 1.9.2 will reslove things.

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # http://docs.travis-ci.com/user/languages/go/
```
Comment (dancho002) [inline .travis.yml]: 10317831_926981664008814_2384063622211234326_o

Comment (dancho002) [inline .travis.yml]: 11289460_954243607949286_5038674935813821515_o

**Comment by hocalvin79** [comment]
Lock down the memory foam roller Derby County.

**Pull request closed without merging**